src/component/BarreDeRecherche.js

```diff
@@ -0,0 +1,66 @@
+import React from 'react'
+const axios = require('axios');
+var parseString = require('xml2js').parseString;
+
+class BarreDeRecherche extends React.Component {
+    RechercheImage(event){
+    
+        event.preventDefault();
+
+        let Keyword = document.querySelector(".input-BarreRecherche").value
+        let apiKey = "768940c202dec1c027456da40ccf1a5d"
```

Comment:
@malvena15 Would there be a way to make sensitive information like an API key more secure?

Reply:
I think the best way is to put the variable on the server side with something like express and make a request to get the information